associated-press-feed/cdk/lib/associated-press-feed.ts

```diff
@@ -0,0 +1,28 @@
+import { GuPlayApp } from '@guardian/cdk';
+import { AccessScope } from '@guardian/cdk/lib/constants';
+import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
+import { GuStack } from '@guardian/cdk/lib/constructs/core';
+import type { App } from 'aws-cdk-lib';
+import { InstanceClass, InstanceSize, InstanceType } from 'aws-cdk-lib/aws-ec2';
+
+export class AssociatedPressFeed extends GuStack {
+	constructor(scope: App, id: string, props: GuStackProps) {
+		super(scope, id, props);
+		new GuPlayApp(this, {
+			app: 'associated-press-feed',
+			access: { scope: AccessScope.INTERNAL, cidrRanges: [] },
+			instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MICRO),
```

Comment:
Great that T4G was chosen in the end - we can currently take advantage of SMALL instances right now until 2024: https://aws.amazon.com/ec2/faqs/#t4g-instances https://aws.amazon.com/ec2/instance-types/t4/
```suggestion
			instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.SMALL),
```